### PR TITLE
Adding analysis of crash ping submission delays

### PR DESCRIPTION
**Pull request by chutten**
Second verse, much like the first.

### CODE REVIEW

File: projects/crash_ping_delays.kp/knowledge.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+- crash ping
+- delay
+created_at: 2017-01-26 00:00:00
+updated_at: 2017-01-26 13:08:54.084263
+tldr: How long does it take before we get crash pings from users in each channel?
+thumbnail: images/output_20_1.png
+---
+### Crash Ping Submission Delays by Channel
+
+This is follow-up analysis to the [Main Ping Submission and Recording Delays by Channel](https://github.com/mozilla/mozilla-reports/blob/master/projects/ping_delays.kp/knowledge.md) analysis previously performed.
```
Comment (harterrt) [inline projects/crash_ping_delays.kp/knowledge.md]: Well, I was going to suggest you point this link to the rtmo version of this report, but it appears that rtmo is currently down.

Comment (vitillo) [inline projects/crash_ping_delays.kp/knowledge.md]: rtmo is back online

Comment (chutten) [inline projects/crash_ping_delays.kp/knowledge.md]: rtmo is a thing that exists?! Cool.

**Comment by chutten** [comment]
Actually, hold up. I think there _might_ be a way to get a recording delay out of crash pings after all... the time from when the crash happened to the time when the ping is created is non-zero, and there might be information present that I can use to measure it.

**Comment by chutten** [comment]
So unfortunately the only time info on a crash ping is crashDate, with a resolution of one day (24hours)

This kinda sucks for my otherwise-smooth CDFs... and it means I should rerun the analysis accounting for it. Yay.

**Pull request closed without merging**